9_snip24_token/main.js

```diff
@@ -0,0 +1,187 @@
+const { Wallet, SecretNetworkClient, EncryptionUtilsImpl, fromUtf8, MsgExecuteContractResponse } = require("secretjs");
+
+const fs = require("fs");
+
+// Load environment variables
+require("dotenv").config();
+
+const main = async () => {
+  const wallet = new Wallet(process.env.MNEMONIC);
+
+  // Create a connection to Secret Network node
+  // Pass in a wallet that can sign transactions
+  // Docs: https://github.com/scrtlabs/secret.js#secretnetworkclient
+  const txEncryptionSeed = EncryptionUtilsImpl.GenerateNewSeed();
+  const secretjs = new SecretNetworkClient({
+    url: process.env.SECRET_LCD_URL,
+    wallet: wallet,
+    walletAddress: wallet.address,
+    chainId: process.env.SECRET_CHAIN_ID,
+    txEncryptionSeed: txEncryptionSeed
```

Comment:
This is unnecessary if not reused in a later session

Reply:
Got it